mmv1/products/secretmanager/Secret.yaml

```diff
@@ -67,6 +72,23 @@ properties:
 
       No more than 64 labels can be assigned to a given resource.
 
+      An object containing a list of "key": value pairs. Example:
+      { "name": "wrench", "mass": "1.3kg", "count": "3" }.
+  - !ruby/object:Api::Type::KeyValuePairs
```

Comment:
I'm seeing some linting errors on this file, I'd guess because of the extra spaces in the description here but I can't tell exactly. Can you try to run the linter and change this so it passes?

Reply:
Ah, found the errors:

repo/mmv1/products/secretmanager/Secret.yaml
  Error: 81:1 [trailing-spaces] trailing spaces
  Error: [8](https://github.com/GoogleCloudPlatform/magic- [trailing-spaces] trailing spaces
  Error: 8[9](https://github.com/GoogleCloudPlatform/magic- [trailing-spaces] trailing spaces
  Error: 91:1 [trailing-spaces] trailing spaces

Reply:
I've removed extra spaces in this file.